src/rust/engine/process_execution/src/remote/streaming.rs

```diff
@@ -52,6 +53,7 @@ pub struct StreamingCommandRunner {
   channel: grpcio::Channel,
   env: Arc<grpcio::Environment>,
   execution_client: Arc<bazel_protos::remote_execution_grpc::ExecutionClient>,
+  overall_deadline_secs: u64,
```

Comment:
Consider taking a `Duration` in your interface to avoid the need to polish notation things.

It would be good for this to be aligned with the existing `queue_buffer_time`...? But not a blocker.

Reply:
Switched to Duration.